src/interaction/ConstrainCOM.py

```diff
@@ -0,0 +1,107 @@
+#  Copyright (C) 2016
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of ESPResSo++.
+#  
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+
+r"""
+***********************************************
+**espressopp.interaction.ConstrainCOM**
+***********************************************
+This class is for calculating forces of constraining center of mass of subchains.
+Subchains are defined as a tuple list.
+
+.. math:: U = k_{com} \left(r - R_{com}\right)^2,
+
+where :math:`R_{com}` stands for the desired center of mass of subchain.
+
+This class set 3 conditions on a tuple list defining subchains.
+
+1. The length of all tuple must be same.
+2. Particles in one tuple must be in a same or neighbor cell list.
+3. The ratio of (Particle ID / The length of a tuple) must not be redundantly.
+
+.. function:: espressopp.interaction.ConstrainCOM(k_com)
+
+		:param k_com: (default: 100.)
+		:type k_com: real
+
+.. function:: espressopp.interaction.FixedLocalTupleListConstrainCOM(system, tuplelist, potential)
+
+		:param system: 
+		:param tuplelist: 
+		:param potential: 
+		:type system: 
+		:type tuplelist: 
+		:type potential: 
+
+.. function:: espressopp.interaction.FixedLocalTupleListConstrainCOM.getPotential()
+
+		:rtype: 
+
+.. function:: espressopp.interaction.FixedLocalTupleListConstrainCOM.setCom(particlelist)
+
+		:param particlelist: 
+		:type particlelist:
+
+
+"""
+from espressopp import pmi, infinity
+from espressopp.esutil import *
+
+from espressopp.interaction.Potential import *
+from espressopp.interaction.Interaction import *
+from _espressopp import interaction_ConstrainCOM, interaction_FixedLocalTupleListConstrainCOM
+
+class ConstrainCOMLocal(PotentialLocal, interaction_ConstrainCOM):
+    
+    def __init__(self, k_com=100.):
```

Comment:
In the reference of this method "Equilibration of high molecular weight polymer melts: A hierarchical strategy, Macro Lett., 2014, 3, 198", $k_com$ is set as 100.
Additionally, $k_com=100$ is strength enough for constraining the center of mass.